lib/MusicBrainz/Server/Form/Field/ArtistCredit.pm

```diff
@@ -123,7 +123,14 @@ sub json {
     my $artists = $c->model('Artist')->get_by_ids(map { $_->{artist}->{id} } @$names);
     for my $name (@$names) {
         $name->{artist}->{entityType} = 'artist';
-        $name->{artist}->{gid} = $artists->{$name->{artist}->{id}}->gid if $artists->{$name->{artist}->{id}};
+        my $artist = $artists->{$name->{artist}->{id}};
+        if ($artist)
+        {
+            $name->{artist}->{gid} = $artist->{gid};
+            $name->{artist}->{sortName} = $artist->{sort_name};
+            $name->{artist}->{comment} = $artist->{comment};
+            $name->{artist}->{editsPending} = $artist->{edits_pending};
```

Comment:
Ok. At first glance Artist JSON seemed to output a lot more details than what is useful to display Artists Credit, but I will look into it again. Thanks for the blessed tip!